src/views/dialogs/FunkitDepositDialog.tsx

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from 'react';
+
+import { useFunkitBuyNobleUsdc } from '@/hooks/useFunkitBuyNobleUsdc';
+
+export const FunkitDepositDialog = () => {
```

Comment:
Since this dialog is just used to launch funkit, I'm thinking we don't need to make a new kind of dialog, instead just use `DepositDialog` and add the logic there to determine to launch Funkit or not. That way other part of the code just need to keep using `openDialog(DialogTypes.Deposit())`

Reply:
Sounds good, will do!

Reply:
Updated in commit https://github.com/dydxprotocol/v4-web/pull/982/commits/e5849d11928c75caa1684c5ca50dcac32715f044! Had to additionally create a deposit slice to manage the new state within the deposit modal